src/lib.rs

```diff
@@ -22,12 +22,28 @@
 //! safety, including ownership and lifetime, for financial and/or cryptographic
 //! software.
 //!
+//! ## Available feature flags
+//!
+//! * `std` - the usual dependency on `std` (default).
+//! * `secp-recovery` - enables calculating public key from a signature and message.
+//! * `base64` - (dependency), enables encoding of PSBTs and message signatures.
+//! * `fuzztarget` - used for testing (fuzzing) only.
```

Comment:
We should drop `fuzztarget` as a feature and instead use `cfg=fuzzing` just like we've done for `rust-secp256k1` and `bitcoin_hashes`.

Reply:
Should it be in this PR or in a separate one?

Reply:
Either is fine, I think.

Reply:
OK, a separate one feels better to me.

Reply:
Alright, can we do that first? Feels a bit strange to document something we plan to drop in the same release :)

Reply:
Done. :)